core/README.md

```diff
@@ -2,23 +2,77 @@
 
 - [STAC API - Core Specification](#stac-api---core-specification)
   - [Recommended Link Relations at `/`](#recommended-link-relations-at-)
+  - [Example Landing Page for STAC API - Core](#example-landing-page-for-stac-api---core)
 
 - **OpenAPI specification:** [openapi.yaml](openapi.yaml) describes the core endpoints ([rendered version](https://api.stacspec.org/v1.0.0-beta.1/core)),
   and [commons.yaml](commons.yaml) is the OpenAPI version of the core [STAC spec](../stac-spec) JSON Schemas.
 - **Conformance URI:** <https://api.stacspec.org/v1.0.0-beta.1/core>
 - **Extension [Maturity Classification](../extensions.md#extension-maturity):** Pilot
 - **Dependencies**: None
 
-The base of a STAC API is its landing page. This resource is the starting point to discover what behaviors the API supports via link relations. 
-This behavior in a RESTful API is known as Hypermedia as the Engine of Application State (HATEOAS). STAC API relies heavily on hypermedia for API endpoint
-navigation. 
+The base of a STAC API is its landing page. This resource is the starting point to discover what behaviors the API supports via the `conformsTo` values and link relations. 
```

Comment:
This isn't as large of a change as the diff implies -- I moved a couple sections around.